web/README.md

```diff
@@ -0,0 +1,59 @@
+# User Overrides Web Panel
+Communicate with the vehicle's CAN bus using a web application. Send custom or preset
+messages through the interactive and user-friendly panel.
+
+## Requirements
+The web panel is ran through the Flask web framework, which comes bundled to
+the default Python installation.
+
+In addition, the application depends on several web sockets to communicate
+with the server. Install them using:
+
+```bash
+pip install flask-socketio
+pip install gevent-websocket
+```
+
+Version inter-compatibility is highly limited, which may lead to several difficulties
+while running the web application. Visit the [version compatibility](https://flask-socketio.readthedocs.io/en/latest/#version-compatibility)
+if encountering such errors.
+
+The application uses [can_data/CAN2.dbc](https://github.com/msfrt/PiLogger/blob/web/web/can_data/CAN2.dbc)
+as database to extract specifications of signals and messages.
+
+## Server Set-Up
+In order to set-up the web server, run the following command in the folder
+where the app.py file is located at the RasPi.
+
+```bash
+flask run --host=0.0.0.0
+```
+
+## Usage
+Once set-up, access the web application [here](http://147.92.111.86:5000/).
```

Comment:
This IP is specific to my home address and will change eventually. You can put `<hostname>:5000` instead, and leave it unlinked.